CHANGELOG.md

```diff
@@ -1,3 +1,21 @@
+x.x.x Release notes (yyyy-MM-dd)
+=============================================================
+### Enhancements
+* Added support for Client Resync which automatically will recover the local Realm in case the server is rolled back. This largely replaces the Client Reset mechanism. Can be configured using `Realm.Sync.Configuration.clientResyncMode`. Three modes are available: `'recover'`, `'discard`', and `'manual'` but currently only `'manual'` is supported for query-based sync. Default is `'manual'`. ([#2328](https://github.com/realm/realm-js/issues/2328))
```

Comment:
isn't default value "recover"?

Reply:
If `recover` is default, it is a breaking change (`manual` is like the old client reset). I'll update the doc.

Reply:
Is that really breaking? It should be transparent? Java also has Recover as default?

Reply:
But it seems Cocoa has Manual as the default?